### PR TITLE
CORTX-25651:CSM log shows error after logout REST API request is sent

### PR DESCRIPTION
**Pull request by Pranali04796**
Signed-off-by: Pranali Ugale <pranali.ugale@seagate.com>

# Pull Request
## Problem Statement
-   CSM log shows error after logout REST API request is sent

## Design
-   For Bug, Describe the fix here.
-   For Feature, Post the link for design

-   \[ \] Coding conventions are followed and code is consistent

-   \[ \] Unit and System Tests are added
-   \[ \] Test Cases cover Happy Path, Non-Happy Path and Scalability
-   \[ \] Testing was performed with RPM
-
image
CSM AGENT LOGS: 
1. login request 
2. Iam user creation
3. logout api
image
-   \[ \] Interface change (if any) are documented
-   \[ \] Side effects on other features (deployment/upgrade)
-   \[ \] Dependencies on other component(s)

-   \[ \] JIRA number/GitHub Issue added to PR
-   \[ \] PR is self reviewed
-   \[ \] Jira and state/status is updated and JIRA is updated with PR link
-   \[ \] Check if the description is clear and explained

-   \[ \] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: csm/core/controllers/login.py [hunk 1]
```diff
@@ -70,4 +70,5 @@ async def post(self):
         await self.request.app.login_service.logout(session_id)
         # TODO: Stop any websocket connection corresponding to this session
         Log.info('Session ended')
+        Log.info(f"user_id: {self.request.session.credentials.user_id} successfully logged out")
```
Comment (rohitkolapkar) [inline csm/core/controllers/login.py:73]: can we merge these two logs into one ?  'Session ended' is too generic and not giving information about the user for which session ended etc. and in the logs it is not not looking good. ( refer dev testing screenshot )

File: csm/core/controllers/login.py [hunk 2]
```diff
@@ -48,11 +48,11 @@ async def post(self):
         password = body.get('password', None)
 
         session_id, body = await self.request.app.login_service.login(username, password)
-        Log.debug(f"Obtained session id for {username}")
+        Log.info(f"Obtained session id for {username}")
```
Comment (rohitkolapkar) [inline csm/core/controllers/login.py:51]: for this log statement, debug would be more appropriate log level.
successful login msg is already set to info ( line 55).

**Review by rohitkolapkar — CHANGES_REQUESTED** [review]

File: csm/core/controllers/login.py (hunk 2 above)
Comment (pawankumarsrivastava) [reply csm/core/controllers/login.py:51]: this log statement is not required. Remove it

File: csm/core/controllers/login.py (hunk 1 above)
Comment (pawankumarsrivastava) [inline csm/core/controllers/login.py:73]: Remove the above log at line #72. Not required.

**Review by pawankumarsrivastava — CHANGES_REQUESTED** [review]

File: csm/core/controllers/login.py (hunk 1 above)
Comment (Pranali04796) [reply csm/core/controllers/login.py]: Addressed

Comment (Pranali04796) [reply csm/core/controllers/login.py:71]: removed

**Review by rohitkolapkar — APPROVED** [review]
Please update the screenshot.

**Review by pawankumarsrivastava — APPROVED** [review]

**Comment by pawankumarsrivastava** [comment]
> Please update the screenshot.

+1. @Pranali04796 Please add screenshots of the logs showing no errors.

**Pull request merged**